### DISCUSSION TITLE
Static Items in Shop

### DISCUSSION THREAD

**Discussion by Archenemis** [Malcolm-Q/LC-LateGameUpgrades#571] [Ideas]
My Idea is a bit simple. To go along with the idea of having a changing list of upgrades, some items are only useful or even usable while they're listed. I'm all for customizing the experience when using this mod, I really loved the ability to have a random five upgrades appear that can be bought when they're in stock. I feel like things like that helps to promote the idea of trying out everything or going for more unconventional builds/upgrades.

Basically.. Was gonna fluff things up a bit, but I'm super tired.. if I need to re-explain, I shall, but my idea is to essentially make it so that people can set certain things to always be available in the shop while Random mode is active. Interns being one of the more key purchases from the shop that needs this. Unless Interns is currently in the random shop list, you can't buy it, meaning you can't purchase an extra intern to let your friends back in to play some more. Then when it is in the list, it's only able to be bought and used while it's actively in the shop list.

Just was thinking that it would be nice to make it so that intern is always on the list (Or have an option to do so) and perhaps for the sake of customizing experiences, might hook people up with the option to have other things always appear on the list as well.

If this idea was already suggested, my bad (I did a search for similar and looked before opening this, so I'm pretty sure I saw nothing about this).. I'm pretty sure it's not in game unless I'm just blind and overlooked it. If it is and I'm just being dumb or tired.. Feel free to let me know

**Comment by WhiteSpike** [comment]
I could be mistaken but interns was never in the upgrade store. It's a command by itself which is just a credit sink for revives.
If people want to have upgrades always appear regardless of random, I could work something out later.

**Reply by Archenemis** [reply]
I could have swore interns popped up in the store too... (I know for a fact when you have the settings on for earning the feats or turning in Apparatus for a perk. It can randomly select intern... Which doesn't really help out at the company building.) But after checking it out, It is its own thing. It just doesn't show up in the list anywhere (That I can see at least). Might be something that could be worked into the menu as well. But it does work as is, it just doesn't say anything about it or indicate it in anyway

Appreciate the fast response on this though
